packages/ui/src/services/api/ApiClient.ts

```diff
@@ -0,0 +1,34 @@
+import { API_LOCATION } from "config"
+import type HttpClient from "./interfaces/httpClient"
+
+enum Method {
+  GET = "GET"
+}
+
+class ApiClient implements HttpClient {
+  readonly jwt: string
+
+  constructor(jwt: string) {
+    this.jwt = jwt
+  }
+
+  async get(route: string): Promise<unknown | Error> {
```

Comment:
```suggestion
  async get(route: string): Promise<unknown> {
```

Get rid of SonarCloud issues